apps/www/src/lib/registry/new-york/ui/calendar/Calendar.vue

```diff
@@ -267,7 +267,7 @@ onMounted(async () => {
   @apply mr-1 text-primary stroke-primary;
 }
 .vc-time-select-group select {
-  @apply bg-transparent p-1 appearance-none outline-none text-center;
+  @apply bg-primary-foreground p-1 appearance-none outline-none text-center;
```

Comment:
@sadeghbarati The problem is that by doing `bg-primary-foreground` instead of transparent, it broke the edges:
![CleanShot 2024-01-12 at 17 40 07@2x](https://github.com/radix-vue/shadcn-vue/assets/20521112/19bac4bb-8f99-40dc-ab2b-d755eb09445d)